src/jit/jitconfig.cpp

```diff
@@ -45,7 +45,7 @@ void JitConfigValues::MethodSet::initialize(const wchar_t* list, ICorJitHost* ho
     {
         // Failed to convert the list. Free the memory and ignore the list.
         host->freeMemory(reinterpret_cast<void*>(const_cast<char*>(m_list)));
-        m_list = "";
+        m_list = nullptr;
```

Comment:
It is often, it can't convert if utf8ListLen == 0, that is empty config string case.